### ISSUE TITLE
Organization logo

### ISSUE THREAD

**Issue by nox** [image-rs/organization#3]
What do you people think of the Rust wheel and a television test pattern inside?

https://fr.wikipedia.org/wiki/Mire_(télévision)#/media/File:Philips_PM5544.svg

**Comment by nox** [comment]
[rust-wheel.svg.zip](https://github.com/image-rs/organization/files/3084131/rust-wheel.svg.zip)

**Comment by 197g** [comment]
What's the licensing situation on that? Otherwise, I like the idea and we have alternatives on what to place inside the wheel, such as:

https://pixabay.com/de/vectors/film-kino-negativ-rahmen-kamera-296751/
https://pixabay.com/de/illustrations/contour-picture-photocamera-3780328/

Licensing: free for non-commercial use, no attribution required.

**Comment by nox** [comment]
AFAIK the wheel is [ok to use](https://www.rust-lang.org/policies/media-guide), and the TV pattern is CC BY 2.5.

**Comment by padenot** [comment]
Here is a first try, from the image linked in the first comment and the image linked in the second comment.

I've added a faint grey border on the rust logo because the test pattern has black boxes. The SVG is in the zip, and you can see a preview below.

[image-rs.zip](https://github.com/image-rs/organization/files/3094263/image-rs.zip)
![image-rs](https://user-images.githubusercontent.com/270485/56359270-5dc15d00-61e1-11e9-869a-c4a79cf4238e.png)

**Comment by nox** [comment]
I like it.

**Comment by 197g** [comment]
Wait, the Rust wheel is not symmetrical.. smh :smiley: The white border looks quite good. I wonder if it would be better for the the small holes to not show the background? The `servo` logo has an interesting interplay between foreground and background here but for the abstract shapes of the TV test image it might work better to not have these small cavities with high complexity.

**Comment by birktj** [comment]
I like the test pattern idea, however I think it has to be simplified a bit to work as a logo, the way it is now I think there are a little too much colors and shapes. Another idea that I experimented a little bit with is to combine the rust wheel with the standard image icon with a mountain and a sun.

**Comment by 197g** [comment]
Based on the simplified film strip pictogram, a little recoloring to make it more consistent. I still find it fairly complex, and maybe the coloring could be even better. The spacing may not be optimal as well.

![image-rs-wheel svg](https://user-images.githubusercontent.com/5550310/56365315-11c1e880-61e0-11e9-8bad-b631e9dc786e.png)

**Comment by birktj** [comment]
Here are some different ideas that I have experimented with.
![image](https://user-images.githubusercontent.com/2932651/56365218-987ec180-61f0-11e9-8be3-d41eb888689c.png)

**Comment by 197g** [comment]
@birktj I really like the bottom right one: It has some nice color, which would be appropriate for an image group if any, but isn't too forward with it and still very orderly.

**Comment by ArturKovacs** [comment]
Here is my entry.

EDIT
Here is the SVG along with a rasterized preview.

[image-rs.svg.zip](https://github.com/image-rs/organization/files/3229586/image-rs.svg.zip)

![image-rs svg](https://user-images.githubusercontent.com/8320264/56396116-b1619400-623d-11e9-88d0-2b808269cfb9.png)

**Comment by 197g** [comment]
It's been a month since then and I'd like to decide on this rather sooner than later. If anyone has other proposals then please bring them forth until next Sunday. I'd also like to encourage submitting constructive critique on those submitted [in the community chat](https://gitter.im/image-rs/community), for example to make one proposal even better or the seek the raw files to edit yourself.

**Comment by 197g** [comment]
Reviving this topic :smiley: I had [someone (on fiverr)](https://www.fiverr.com/aishdesigner) do three concepts. The offer comes with the possibilities of revisions so your feedback, on color, visual details, other to each of them is appreciated. First of all, the concepts below:

![design-0](https://user-images.githubusercontent.com/5550310/83539693-de771180-a4e6-11ea-9503-ec5b4f4d0f51.jpg)
![design-1](https://user-images.githubusercontent.com/5550310/83539694-df0fa800-a4e6-11ea-8ea0-22fbede92039.jpg)
![design-2](https://user-images.githubusercontent.com/5550310/83539695-dfa83e80-a4e6-11ea-8b91-9501e1995cef.jpg)

**Comment by ArturKovacs** [comment]
Out of these three I think the third one has the most potential but its colors need to be a bit desaturated in my opinion.

This also inspired me to update my own entry so here goes v2.0

EDIT Just to be clear the SVG and the PNG below are licenced under the [CC0 1.0](https://creativecommons.org/publicdomain/zero/1.0/) license. In other words they are in the public domain.

[image-rs-v2.0.svg.zip](https://github.com/image-rs/organization/files/4718337/image-rs-v2.0.svg.zip)

![image-rs](https://user-images.githubusercontent.com/8320264/83542803-c65dbd00-a4fb-11ea-993c-f53b705243c9.png)

**Comment by 197g** [comment]
> Out of these three I think the third one has the most potential but its colors need to be a bit desaturated in my opinion.

Then you're the second one with that immediate reaction, I thought the exact same thing but didn't want to influence the first feedback :)

**Comment by kaj** [comment]
I think that a logo should look good in a large view, of course, but equally important and harder to accomplish is that it needs to both look good and be recognizable at small size.

For that reason, I think @ArturKovacs rust-brown shutter and @birktj s first, the colourful gear, are the best suggestios I've seen in this thread yet.

**Comment by 197g** [comment]
I like the simplicity in @ArturKovacs suggestion. But I also really like playing mostly with primaries for an org focussed on image processing as a whole and not just the camera portion of it. I've forwarded the feedback on saturation and got (better) coloring back, see below. (Still prefer the first design as a whole).

![logo design image rs-01](https://user-images.githubusercontent.com/5550310/84423413-f0a32f00-ac0d-11ea-9dea-a3688349d2de.png)
![logo design image rs-04](https://user-images.githubusercontent.com/5550310/84424099-1e3ca800-ac0f-11ea-85bb-ad2c53e57531.png)

**Comment by ArturKovacs** [comment]
Yeah I also still prefer the first one. But I think that it looks odd that the wholes on the wheel are above the frame when the wheel itself is below it. I think that the frame shouldn't appear to have wholes in it.

Also I would like to see a version where the wheel is visible underneath the frame at the right side. I know that it's on purpose but it gives the impression that the cogwheel itself thins out.

**Comment by 197g** [comment]
Taking in all the feedback and my own nits: Here's a cleaned up version with an adjustment to the color palette—the wheel is more faithful to the Rust logo and pure black. With the corrected alignment of the top frames the wholes of the wheel look far less weird, ensuring that the gear wheel is visible is an improvement so thanks for point it out.

![image-rs-logo-2](https://user-images.githubusercontent.com/5550310/85436696-f804f980-b578-11ea-9758-1ab0ab91ec80.png)

[image-rs-logo.zip (svg)](https://github.com/image-rs/organization/files/4821065/image-rs-logo.zip)

> I think that a logo should look good in a large view, of course, but equally important and harder to accomplish is that it needs to both look good and be recognizable at small size.

That's something I thought about for a while. So far `image-rs` as an org shouldn't really require a small logo. Recognizability at a distance is a bit different than being size and resolution constrained. Instead, I would rather use the alternatives for specific programs that would benefit from having a small, minimal resolution icon in their desktop metadata. That would also be my rationale for preferring this one above the others: I find it cleaner and less clunky at large sizes.

Also have you tried it with pride gradient? It really works _really_ well imho:

![image-rs-pride](https://user-images.githubusercontent.com/5550310/85441280-731cde80-b57e-11ea-9447-51d155da32c1.png)

(Small sneaky edit: Github's change to the shape of icons is luckily restricted to user icons for now).

**Comment by nabijaczleweli** [comment]
NB: the black one is nigh-invisible on a dark background, whereas the gay one works decidedly better. Confer screenshot: 
![shot](https://user-images.githubusercontent.com/6709544/85475589-9bc0ca80-b5b6-11ea-9c04-155a8c6e755e.png)

**Comment by jplatte** [comment]
Bump, the current GitHub org logo really doesn't look good with the dark theme.
